src/viewer/App.tsx

```diff
@@ -415,9 +443,10 @@ export const App = observer(({store}: { store: AppStateStore }) => {
   }
   const {frames, activeFrame} = connection;
   const active = frames.find(f => f.id === activeFrame);
+  const filteredFrames = searchResult.length ? frames.filter((frame) => searchResult.includes(frame.id)) : frames;
 
-  const Row = ({ index, style, data }: ListChildComponentProps) => {
-    const frame = data[index];
+  const Row = ({ index, style }: ListChildComponentProps) => {
+    const frame = filteredFrames[index];
```

Comment:
@SerCeMan 
I fixed that regression which I caused in my previous PR
https://github.com/rsocket/rsocket-chrome-devtools/pull/19#discussion_r785548093

Reply:
No worries, I'll do another round of testing before releasing the extension.